docs/production-deployment/cloud/metrics/datadog.mdx

```diff
@@ -32,6 +32,13 @@ tags:
 Exporting cloud metrics to Datadog provides enhanced observability, allowing you to monitor, alert, and visualize key performance indicators of your applications and infrastructure.
 Temporal's integration with Datadog extends the monitoring capabilities of your Temporal Cloud deployment.
 
+:::note
+
+This tutorial provides an example application to help users capture metrics from Temporal's PromQL endpoint and export to  DataDog.
```

Comment:
1. Double space on line 37.
2. Consider "This tutorial's [sample app](...) helps you capture..." because it's so indirect the way it's currently worded. It's a reader, not a user, and more specifically it's you, the reader.

Reply:
I believe this is phrased the way it is to emphasize that this is just an unmaintained sample (as I mentioned in my comment, we eventually need to both take over maintenance of this and move it to learn), so this is more of a disclaimer, hence "an example application". I know it's kind of weak language but I might prefer to keep it (and thank you for catching the double space).

Reply:
Works for me.